cirq/google/engine/engine_program.py

```diff
@@ -260,6 +263,56 @@ def get_job(self, job_id: str) -> engine_job.EngineJob:
         return engine_job.EngineJob(self.project_id, self.program_id, job_id,
                                     self.context)
 
+    def list_jobs(self,
```

Comment:
There should probably be another method on the Engine object to list jobs across programs.

Reply:
Got it - that makes sense